src/views/SpaceProposals.vue

```diff
@@ -49,7 +50,10 @@ async function loadProposals(skip = 0) {
   store.space.proposals = store.space.proposals.concat(proposalsObj);
 }
 
-onMounted(load());
+onMounted(() => {
+  setPageTitle('page.title.space.proposals', { space: props.space.name });
```

Comment:
It shows `Proposals - ENS - Snapshot` as title in spaces. I think just `ENS Proposals - Snapshot` would be better.

More space title examples:
`About ENS - Snapshot`
`Create ENS proposal - Snapshot`

@bonustrack what are your thoughts?

Reply:
okay, i will review ite.